tasks/run-cmd.yaml

```diff
@@ -37,7 +37,7 @@ spec:
         ls -la
 
         # Execute the script commands within the Jumpstarter shell
-        jmp client shell <<-EOF
+        jmp shell <<-EOF
```

Comment:
I've looked at the new changes but I'm not sure if we need to add something else here as arguments.

Reply:
Giving it a try, I will confirm in a few minutes.

Reply:
use:    `jmp shell --lease "${JMP_LEASE_ID}"`

We need this for now,  it wasn't necessary in the past, I believe it's a regression.